webui/src/pages/Cluster/Cluster.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import React from 'react';
+import { useStore } from 'effector-react';
+// @ts-ignore
+import { Spin } from '@tarantool.io/ui-kit';
+
+import PageDataErrorMessage from 'src/components/PageDataErrorMessage';
+import { PageLayout } from 'src/components/PageLayout';
+import { cluster } from 'src/models';
+
+import { ClusterControllers } from './ClusterControllers';
+import { ClusterModals } from './ClusterModals';
+import { ClusterPanels } from './ClusterPanels';
+import ButtonsPanel from './components/ButtonsPanel';
+import ReplicasetListPageSection from './components/ReplicasetListPageSection';
+import UnconfiguredServerListPageSection from './components/UnconfiguredServerListPageSection';
+
+const { $clusterPage } = cluster.page;
+
+const CLUSTER_PAGE_TITLE = 'Cluster';
```

Comment:
Maybe need created some exported constant? https://github.com/tarantool/cartridge/pull/1578/files#diff-a5d15763672cabe3ab885070ef25e3e3b8adef632e6a928d84025f2433c12796R13

Reply:
Fixed it a little differently.